test/show.jl

```diff
@@ -269,3 +269,12 @@ end
 @test_repr "bitstype 100 B"
 @test repr(:(bitstype A B)) == ":(bitstype A B)"
 @test repr(:(bitstype 100 B)) == ":(bitstype 100 B)"
+
+oldout = STDOUT
```

Comment:
Should be in `let` block and I don't think the `try` block or even the `@test` is necessary.